data/remnant/remnant 2 side missions.txt

```diff
@@ -114,8 +114,8 @@ mission "Remnant: Visit to Paradise 1"
 				`	"I do not know what to make of this."`
 				`	"This is a spaceport?"`
 			`	She nods, and guides you amongst the stream of camel-drawn hovercarts heading for the gate. "This is one of our earliest spaceports, before we learned new techniques better suited to our new home. Our founding ships brought as much as they could, of course, but as it turns out, the earliest phases of expansion look much the same, whether one is looking at historic pre-flight eras or modern settlement efforts."`
-			scene "scene/remnant remote spaceport"
 			label questionloop
+			scene "scene/remnant remote spaceport"
```

Comment:
Is there a reason for this change?

Reply:
It doesn't make sense for it to be immediately hidden by the choice, the way it would be above this label.

Reply:
Do you have a save file for this?

Reply:
No. I can revert it if needed.

Reply:
Yeah, that's probably the safest bet for now.